libs/util/src/lib/calculate-timeout-timer.ts

```diff
@@ -0,0 +1,20 @@
+import { AVERAGE_WPM } from '@razor/constants';
+
+const averageWPM = AVERAGE_WPM;
+
+/** Calculating timeout timer for the race text.
+ *
```

Comment:
Instead of the comments you've added inside, you can do the following in the function description:
==
Computes the time it would take for an average person to type the given block of text, and
returns it as the maximum allowed duration.

Reply:
Now, when you write up this description, you'll start noticing what's wrong in the approach that has been used to compute the time out duration

Reply:
updated the function 9b960edbf7f516c3679f74a1f1fb1aae9a717cb4